public/give.js

```diff
@@ -0,0 +1,1164 @@
+import toDataByLocation from './toDataByLocation.js';
```

Comment:
Does 'give' really need its own script?

Reply:
I moved it out so that I could focus on index and stuff was starting to break as the filters changed. The one thing I'm worried about with 'give' is that it's already embedded in a bunch of other sites so I want to make sure we don't break anything by adjusting the height of the page too much so I didn't wanna change the filters in the first pass.

Reply:
@drwestco took a stab at consolidating back here: https://github.com/findthemasks/findthemasks/pull/624

We lose out on state filters but with the list synced with the map I think that's OK?